src/Controller/DefaultController.php

```diff
@@ -28,11 +28,11 @@ public function indexAction(\DataDictionaryBundle\Graph\Graph $graph)
         $graphViz = new GraphViz($graph);
 
         return $this->render(
-            "DataDictionaryBundle:Default:index.html.php",
+            "@DataDictionary/index.html.twig",
```

Comment:
'Default' section is no longer needed?

Reply:
No since 3.4 that has been changed. I could add a extra directory but wasnt needed